src/app/home/partners/partners.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+
+import { Observable } from 'rxjs';
+
+import { PartnerService } from 'src/app/core/services/partner.service';
+import { Partner } from 'src/app/core/model/partner';
+
+@Component({
+  selector: 'sw-partners',
+  templateUrl: './partners.component.html',
+  styleUrls: ['./partners.component.scss']
+})
+export class PartnersComponent implements OnInit {
+
+  partners = new Observable<Partner[]>();
+  
+  constructor(private partnerService: PartnerService) { }
+
+  loadPartners(): void {
+    this.partners = this.partnerService.getPartners()
+  }
+
+  ngOnInit() {
+    this.loadPartners()
+  }
```

Comment:
Please keep on init function at the beginning. Also, do not forget to always define the return type, to keep consistent across the app. And I don't see the value of having a public function called loadPartners, I don't see it is being used anywhere else, maybe you could just initialize the observable directly here.